packages/prime/src/prime_cli/commands/env.py

```diff
@@ -782,7 +782,7 @@ def pull(
         if target:
             target_dir = Path(target)
         else:
-            target_dir = Path.cwd() / f"{owner}-{name}-{version}"
```

Comment:
This gives it more of a git-like feel, so I `prime env pull will/wordle` into the `wordle` dir for development